docs/variable-management.md

```diff
@@ -48,3 +48,40 @@ For example, this is how variables are managed in this project:
 * for the ML variables that need to change for experimentations, define them as [ML pipeline variables](../samples/image-classification-tensorflow/ml_service/pipelines/build_training_pipeline.py#L46). The defaults for ML variables are stored in [ml_model/parameters.json](../samples/image-classification-tensorflow/ml_model/parameters.json) so that they can run in DevOps pipelines [without having to be specified in the pipeline yamls](../samples/image-classification-tensorflow/devops_pipelines/03-train-evaluate-register-model.yml#L119).
+
+# Example
+Here are variable examples for yesno sample in `samples\yesno` directory
+
+## Key vault secrets
+| Variable Name            | Short description                                                                    |
+| ------------------------ | -------------------------------------------------------------------------------------|
+| ACR_PASSWORD             | Azure Container Registory user password                                              |
+| AML_STORAGE_ACCOUNT_KEY  | Azure Storage Account Key where input data is stored                                 |
+
+## Infrastructure variables
+| Variable Name            | Short description                                                                    |
+| ------------------------ | -------------------------------------------------------------------------------------|
+| BASE_NAME                | Unique naming prefix for created resources - max 10 chars, letters and numbers only  |
+| LOCATION                 | Azure location where Azure resources are deployed                                    |
+| RESOURCE_GROUP           | Azure Resource Group name                                                            |
+| AZURE_RM_SVC_CONNECTION  | Azure Resource Manager Service Connection name                                       |
+| SUBSCRIPTION_ID          | Azure Subscription ID for the project                                                |
+| WORKSPACE_NAME           | Azure Machine Learning workspace name                                                |
+
+## Application variables
+| Variable Name            | Short description                                                                    |
+| ------------------------ | -------------------------------------------------------------------------------------|
+| KEYVAULT_NAME            | Key vault name where storage account key, Container registry password are stored     |
```

Comment:
Key Vault is still infrastructure, right?

Reply:
Actually, KEYVAULT_NAME is generated with `BASE_NAME`, and I deleted it and let Ken change yaml files. So, we do not need KEYVAULT_NAME here any more